src/xcode/ENA/ENA/Source/Scenes/ENSetting/ExposureNotificationSettingViewController.swift

```diff
@@ -318,12 +318,12 @@ final class ExposureNotificationSettingViewController: UITableViewController, Ac
 	}
 
 	private func daysSinceInstallationCell(for indexPath: IndexPath, in tableView: UITableView) -> UITableViewCell {
-		guard let daysSinceInstallationCell = tableView.dequeueReusableCell(withIdentifier: ReusableCellIdentifier.daysSinceInstallationCell.rawValue, for: indexPath) as? DaysSinceInstallTableViewCell else {
+		guard let daysSinceInstallationCell = tableView.dequeueReusableCell(withIdentifier: ReusableCellIdentifier.daysSinceInstallationCell.rawValue, for: indexPath) as? DaysSinceInstallTableViewCell,
+			  let appInstallationDate = store.appInstallationDate else {
 			fatalError("Cell is not registered")
```

Comment:
I think we should not use a fatalError if the `appInstallationDate` is not set, maybe use a separate constant that contains something like `let installationAgeInDays = store.appInstallationDate.map { Calendar.autoupdatingCurrent.startOfDay(for: $0).ageInDays } ?? 0` and then pass this into the `max()` call below

Reply:
I''ll try the suggested .map solution.